src/ui/Quickview/Quickview.ts

```diff
@@ -453,10 +464,64 @@ export class Quickview extends Component {
         body: this.element.ownerDocument.body,
         sizeMod: 'big'
       });
+      this.makeModalboxAccessible(this.modalbox.modalBox);
       return computedModalBoxContent;
     });
   }
 
+  private getFocusableElements(container: HTMLElement) {
+    return _.sortBy(container.querySelectorAll<HTMLElement>('[tabindex]'), element => element.tabIndex);
+  }
+
+  private createFocusTrap(element: HTMLElement) {
+    document.addEventListener('focusin', e => {
+      if (!this.modalbox) {
+        return;
+      }
+      if (!element.contains(e.target as HTMLElement)) {
+        const focusableElements = this.getFocusableElements(element);
+        focusableElements[0].focus();
+        e.preventDefault();
+        return;
+      }
+    });
+    $$(document.body).on('keydown', (e: KeyboardEvent) => {
+      if (!this.modalbox) {
+        return;
+      }
+      if (e.keyCode !== KEYBOARD.TAB) {
+        return;
+      }
```

Comment:
I'm worried this might not work on some screen readers. Is there some tools I should test this with?

Reply:
You can use osx voiceover.